doc/running.rst

```diff
@@ -163,3 +186,30 @@ To generate code that is compatible with particular release versions of NEST Sim
 - The default is the empty string, which causes the NEST version to be automatically identified from the ``nest`` Python module.
 - ``"v2.20.2"``: Latest NEST 2 release.
 - ``"master"``: Latest NEST GitHub master branch version (https://github.com/nest/nest-simulator/).
+
+
+Python-standalone target
+------------------------
+
+*NESTML features supported:* :doc:`neurons </nestml_language/neurons_in_nestml>`
+
+The aim of the Python-standalone target is to facilitate model development and debugging. A Python class is generated for each neuron, as well as a very simple simulator that applies some spikes to the model(s) and measures the results. This generated code can be run independently of any installed simulator (only a few common Python packages are required, like scipy for numerical integration). The generated Python code is intended to be easy to read and understand, rather than to be fast. When satisfied with the Python target results, high-performance code can then be generated by simply switching to a different target platform.
+
+.. list-table::
```

Comment:
Could you also add how to run this? I suspect it's just `python test_python_standalone_module.py`?

Reply:
I added a shell command example just after this paragraph, could you have a look? Cheers!

Reply:
This looks good to me. Thanks!